api.js

```diff
@@ -1801,7 +1803,114 @@ module.exports = (function () {
       });
     });
 
+    app.get(padRoute + '/getLastEdited/:key', function (req, res) {
+      var utils = require('./utils');
+      var etherpadAPI = require('ep_etherpad-lite/node/db/API');
+
+      var padId = req.params.key; 
+
+      return etherpadAPI.getLastEdited(padId).catch(err => {
     
+        return {padId}; 
```

Comment:
By implementing it this way, in case an error is encountered, it just responds with a padId.  It would be better if you can send the error to the user.